packages/docs/site/i18n/es/code.json

```diff
@@ -0,0 +1,432 @@
+{
+	"theme.ErrorPageContent.title": {
+		"message": "This page crashed.",
+		"description": "The title of the fallback page when the page crashed"
```

Comment:
I assume that these will be translated later.

Reply:
Yes, I have left the file prepared to be translated.
This PR is just focused on the initial setup for i18n so other contributors can start translating Playground docs pages